packages/accordion/demo/index.html

```diff
@@ -49,28 +49,28 @@ <h2 class="accordion-label">Accordion Heading</h2>
 
 		</div> <!-- //.accordion -->
 
-		<div class="accordion">
-			<button id="header3" class="accordion-header" type="button">Accordion Header</button>
-			<div class="accordion-content">
-				<h2 class="accordion-label">Accordion Heading</h2>
+		<div class="custom-accordion">
```

Comment:
@Firestorm980 

I get changing this, but I'm wondering if a more relevant namespace would be more appropriate than "custom" (i.e. `baseline-`).

Also, looking at this, should we be more BEM?

Reply:
@joesnellpdx I don't mind what the class is here, since it is just an example. `custom-` just seemed to help drive the point a little more to me at the time (it could've just as easily been `my-custom-class-`). If we want to have an example with classes that are more 10up standard (like with BEM), I think that's fine.